Document/0x05b-Basic-Security_Testing.md

```diff
@@ -777,21 +778,52 @@ This will install any module that matches your query. Newly installed modules ar
 
 #### Potential Obstacles
 
-Discuss with your project team the possibility of providing a debug build for the following security controls, which may be implemented in the app you're about to test. A debug build provides several benefits for a (white box) test by allowing a more comprehensive analysis.
+Applications often implement security controls that make it more difficult to perform a security review of the application, such as root detection and Certificate pinning. Ideally, you would acquire both a version of the application that has these controls enabled, and one where the controls are disabled. This allows you to analyze the proper implementation of the controls, after which you can continue with the less-secure version for further tests.
+
+Of course, this is not always possible, and you may need to perform a black-box assessment on an application where all security controls are enabled. The section below shows you how you can circumvent Certificate pinning for different applications.
 
 ##### Certificate Pinning
 
-If the app implements certificate pinning, X.509 certificates provided by an interception proxy will be declined and the app will refuse to make any requests through the proxy. To perform an efficient white box test, use a debug build with deactivated certificate pinning.
+Different ways of implementing Certificate Pinning have been explained in "Testing Custom Certificate Stores and Certificate Pinning". 
```

Comment:
"Certificate Pinning" should be written consistently. Either "Certificate Pinning" or "certificate pinning". I guess "Certificate Pinning" makes more sense as it should be considered as a proper noun.

Reply:
I've renamed to certificate pinning everywhere now, except for in the reference above to a title, which as to be capitalised.